spec/features/user_signs_up_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+feature 'User signs up', %Q{
+  As a prospective user
+  I want to create an account
+  So that I can track my medications and medical history
+} do
+
+  # Acceptance Criteria:
+  # - There is a link to "sign up"
+  # - The sign up page has a form for:
+  #   - First Name
+  #   - Last Name
+  #   - Email
+  #   - Password
+  #   - Password Confirmation
+  # - The form requires me to provide all information
+  # - I am logged in once I sign up
+
+  scenario "there is a link for user to sign up" do
+    visit root_path
+
+    expect(page).to have_content "Sign Up"
+  end
+
+  scenario "sign up form is provided" do
+    visit new_user_registration_path
+
+    expect(page).to have_content "First Name"
+    expect(page).to have_content "Last Name"
+    expect(page).to have_content "Date of Birth"
+    expect(page).to have_content "Email"
+    expect(page).to have_content "Password"
+    expect(page).to have_content "Password confirmation"
+  end
```

Comment:
This test file looks great! I'd even go so far as to say that you don't actually *need* these first two scenarios here ^ since in your test right below this comment, you're actually implicitly checking to make sure that the "Sign Up" link and the form fields are all there, since you're actually using them during the actions of the test. So if you wanted to make this file shorter, I think you'd be totally fine to just keep the two `scenario`s below this comment, and delete the two above, but leaving them all in is also fine!